<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Systemset</class>
 <widget class="QWidget" name="Systemset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>426</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>114</red>
        <green>137</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>114</red>
        <green>137</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>114</red>
        <green>137</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>114</red>
        <green>137</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="SystemLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Jur_And_User">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>权限/用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="DateTime">
   <property name="geometry">
    <rect>
     <x>574</x>
     <y>10</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>2017/02/01 13:28</string>
   </property>
  </widget>
  <widget class="QPushButton" name="power">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>权限</string>
   </property>
  </widget>
  <widget class="QPushButton" name="language">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>语言</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>自检</string>
   </property>
  </widget>
  <widget class="QPushButton" name="screen_display">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>97</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>屏幕与显示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="graphics_clear">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>图形清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="network">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>联网</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setting">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>117</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>恢复出厂设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="parameter">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>设备参数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="in_out">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>输入输出</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>701</width>
     <height>251</height>
    </rect>
   </property>
   <widget class="QWidget" name="Graphicswidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #000;</string>
    </property>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#D4E6CA;
border-radius:3px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>图形累计数:</string>
     </property>
    </widget>
    <zorder>spinBox</zorder>
    <zorder>label</zorder>
    <zorder>Screenwidget</zorder>
   </widget>
   <widget class="QWidget" name="Screenwidget" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>171</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #000;</string>
    </property>
    <widget class="QPushButton" name="adjustment">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>校准</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dataset">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>日期</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ligntopen">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>亮度</string>
     </property>
    </widget>
    <widget class="QPushButton" name="screenprotect">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>屏保</string>
     </property>
    </widget>
    <zorder>adjustment</zorder>
    <zorder>dataset</zorder>
    <zorder>ligntopen</zorder>
    <zorder>screenprotect</zorder>
    <zorder>widget</zorder>
   </widget>
   <widget class="QWidget" name="Languagewidget" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #000;</string>
    </property>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>89</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>中文</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>89</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>英文</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Powerwidget" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #000;</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>89</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>操作员</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>89</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>管理员</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>89</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>制造商</string>
     </property>
    </widget>
    <zorder>radioButton</zorder>
    <zorder>radioButton_2</zorder>
    <zorder>radioButton_3</zorder>
    <zorder>Languagewidget</zorder>
    <zorder>widget</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
